src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -0,0 +1,77 @@
+package nextstep.subway.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Embeddable
+public class Sections {
+
+    private static final int FIRST_SECTION_INDEX = 0;
+    private static final String NOT_EXIST_SECTIONS_EXCEPTION = "삭제할 Sections이 존재하지 않습니다.";
+    private static final String NOT_SAME_DOWN_STATION_EXCEPTION = "마지막 구간의 하행종점역이 삭제할 하행종점역과 일치하지 않습니다";
+
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
+    protected Sections() {
+    }
+
+    public Sections(List<Section> sections) {
+        this.sections = sections;
+    }
+
+    public void add(Section section) {
+        this.sections.add(section);
+    }
+
+    public List<Station> getStations() {
+        if (sections.isEmpty()) {
```

Comment:
👀  중복이 발생한 것 같아요. 하지만 해당 메서드 이름을 쓰면 `getStations` 에서는 조금 어색하다고 생각될 수 있겠네요

```suggestion
        if (isInValidSize()) {
```